omod/src/test/java/org/openmrs/module/idgen/web/RestTestConstants.java

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
why did you add this?

Reply:
I think I've added these files before I handed the ticket to you. I'll address them

Reply:
Remove them it's not needed